doc/src/__latexindent_temp.tex

```diff
@@ -0,0 +1,11 @@
+\author{
+\textbf{\textit{Project Manager:}}
+ \\Cristian Rabiti\\
+ \textbf{\textit{Principal Investigator and Technical Leader:}}
+\\Paul W. Talbot\\
+\textbf{\textit{Main Developers:}}
+\\Paul W. Talbot\\
+Abhinav Gairola\\
+\textbf{\textit{Additional  Developers:}}
+\\Jia Zhou
```

Comment:
idk what this file is, but it seems to have wrong information. Maybe it shouldn't be tracked?